<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mercurial::Internal::SrcDestDialog</class>
 <widget class="QDialog" name="Mercurial::Internal::SrcDestDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QRadioButton" name="defaultButton">
       <property name="text">
        <string>Default Location</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="localButton">
       <property name="text">
        <string>Local filesystem:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="urlButton">
       <property name="toolTip">
        <string>For example: 'https://[user[:pass]@]host[:port]/[path]'.</string>
       </property>
       <property name="text">
        <string>Specify URL:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="Utils::PathChooser" name="localPathChooser" native="true">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="urlLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>For example: 'https://[user[:pass]@]host[:port]/[path]'.</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="defaultPath">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="promptForCredentials">
         <property name="text">
          <string>Prompt for credentials</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::PathChooser</class>
   <extends>QWidget</extends>
   <header location="global">utils/pathchooser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Mercurial::Internal::SrcDestDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Mercurial::Internal::SrcDestDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>urlButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>urlLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>localButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>localPathChooser</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
